python-squid/src/main/java/org/sonar/python/PythonConfiguration.java

```diff
@@ -20,16 +20,18 @@
 package org.sonar.python;
 
 import java.nio.charset.Charset;
-import org.sonar.squidbridge.api.SquidConfiguration;
 
-public class PythonConfiguration extends SquidConfiguration {
+public class PythonConfiguration {
 
+  private Charset charset;
+  private boolean stopSquidOnException = false;
   private boolean ignoreHeaderComments;
 
   public PythonConfiguration(Charset charset) {
-    super(charset);
+    this.charset = charset;
   }
 
+  // TODO this method seems to be unused, should we create plugin property to ignore header comments?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)